src/PhpSpreadsheet/Shared/StringHelper.php

```diff
@@ -560,6 +560,22 @@ public static function convertToNumberIfFraction(string &$operand): bool
 
     //    function convertToNumberIfFraction()
 
+    /**
+     * Identify whether a string contains a percentage, and if so,
+     * convert it to a numeric.
+     *
+     * @param string $operand string value to test
+     */
+    public static function convertToNumberIfPercent(string &$operand): bool
+    {
+        if (strpos($operand, '%', -1) !== false) {
```

Comment:
Perhaps this check should be tightened up, because a string that contains only `'%'` will return a true here, even with no actual numeric value to convert. MS Excel will treat this as a string, and it won't be treated as 0.0%, so it will give a `#VALUE!` error if used in an arithmetic formula.

Reply:
And MS Excel also treats a string containing a leading percentage sign (e.g. `'%2.5'`) as a numeric percentage value in numeric calculations

Reply:
There may also be whitespace before a leadingg percentage sign, or after a trailing percentage sign

Reply:
Great feedback, thanks.  Was only considering my particular case.
Just pushed a  commit that switches to a regular expression.  I've also attached an Excel sheet that tests a bunch of different permutations and combinations.
Let me know what you think.
[Excel Test Cases.xlsx](https://github.com/PHPOffice/PhpSpreadsheet/files/9946417/Excel.Test.Cases.xlsx)

Also just realized that I'm working on `master` on my fork instead of a new branch; sorry about that.